lib/microos.pm

```diff
@@ -31,6 +31,11 @@ sub microos_login {
         send_key_until_needlematch 'tty2-selected', $key, 11, 30;
     }
 
+    if (is_ipmi) {
+        reset_consoles;
+        # Give some time for system fully booting and sshd ready
+        sleep 5;
```

Comment:
What problem are you trying to solve here?

Reply:
Sometimes ssh console can't be established successfully, but chance not high. So do some reset and wait here.

Reply:
Agree. I met twice among several tens of verification jobs. The failure is like this -- http://10.67.129.77/tests/297#step/selfinstall/8. I guess sshd is ready but do not know why the screen isn't updated after password typed. I will have a look further and fix in new PR as a separate issue. Let's drop this code piece now and let this PR focus on the sle micro 6 baremetal installation via usb.

Reply:
My above comment is not fully right. Actually this code block mainly fixes that after selfinstall, the root-ssh console needs to be reconnected because original console is already broken after reboot, so reset_console then select_console. After more thoughts, I will move it to usb_install.pm after power reset, which is better.

Reply:
DONE